odoo/tools/mail.py

```diff
@@ -290,6 +292,22 @@ def is_html_empty(html_content):
     tag_re = re.compile(r'\<\s*\/?(?:p|div|span|br|b|i|font)(?:(?=\s+\w*)[^/>]*|\s*)/?\s*\>')
     return not bool(re.sub(tag_re, '', html_content).strip())
 
+def parse_inline_template(text):
```

Comment:
Would move to ``jinja.py`` that we could rename ``rendering_tools.py``

Reply:
Done. But there is only the variable `template_env_globals` that is in the file now as the environment is not used anymore.